README.md

```diff
@@ -83,25 +104,23 @@ for some plugins (e.g.: [`serverless-offline`](https://github.com/dherault/serve
 _Note_: with the defaults above `index.js` will be saved, overwriting without
 warning, in the project root (the parent dir of `out`).
 
-### Configuration
+### Exit on compilation warnings
 
-As an alternative to `cljsCompiler: lumo`, `cljsCompiler.lumo` may be specified
-as a map of options.  Currently supported:
+Lumo generates warnings such as `WARNING: Use of undeclared Var` to signal
+failures. You can tune the ones you want to see by using the
+[`:warnings` compiler option](https://clojurescript.org/reference/compiler-options#warnings)
+in `serverless-lumo.edn`, but by default the `lumo` process emits the warnings,
+does not throw and returns `0`. This means that `serverless` will keep going in
+presence of warnings.
 
-```yml
+If you want to stop compilation when the first warning is emitted instead,
+either pass the `--warningExit` option to `package` or add:
+
+```yaml
 custom:
-  cljsCompiler:
-    lumo:
-      dependencies:
-        - andare:0.7.0
-      classpath:
-        - /tmp/
-      localRepo: /xyz
-      cache: /cache | none
+  cljsWarningExit: true
```

Comment:
This is a Lumo-specific parameter, so ideally we store it beneath the `lumo` key in serverless.yml.

Reply:
Uhm, true, does this apply to `index` as well?
How does the `lein` compiler work? Does it fail on warnings at the moment (I assume no)?

Reply:
Yeah, if `index` is still a valid option, then it's lumo-specific.  the jvm compiler doesn't fail on warnings.

Reply:
Ok cool, will change that in the PR as well, so that we keep both in there.